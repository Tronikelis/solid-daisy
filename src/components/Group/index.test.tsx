import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Group } from ".";

describe("<Group />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Group>foo</Group>);
        expect(container).toMatchSnapshot();
    });
});
