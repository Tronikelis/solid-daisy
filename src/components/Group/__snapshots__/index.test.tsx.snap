// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Group /> > renders correctly 1`] = `
<div>
  <div
    class="flex flex-row items-center gap-4"
  >
    foo
  </div>
</div>
`;
